src/api/db/migrate/20170323145236_add_user_id_to_bs_request.rb

```diff
@@ -0,0 +1,21 @@
+class AddUserIdToBsRequest < ActiveRecord::Migration[5.0]
+  def up
+    add_reference(:bs_requests, :user)
+    User.all.pluck(:id, :login,).each do |user|
+      execute "UPDATE bs_requests SET user_id = '#{user[0]}' WHERE creator = '#{user[1]}'"
+    end
```

Comment:
hm, and how long did it take? the ruby loop looks suspecious ...

Can't you do that via a single SQL statement like
 
 UPDATE reqs SET reqs.user_id = u.id FROM bs_requests AS reqs INNER JOIN users u ON reqs.creator = u.login

(100% untested, just typed in ;)